src/containers/add-documents/AddDocuments.tsx

```diff
@@ -0,0 +1,62 @@
+import { FC, useEffect, useState } from 'react'
+
+import Box from '@mui/material/Box'
+
+import FileUploader from '~/components/file-uploader/FileUploader'
+import { styles } from '~/containers/add-documents/AddDocuments.styles'
```

Comment:
move styles to const and remove space between libs

Reply:
Fixed